.github/workflows/self-scheduled-intel-gaudi3-caller.yml

```
@@ -0,0 +1,70 @@
+name: Self-hosted runner (Intel Gaudi3 scheduled CI caller)
+
+on:
+  repository_dispatch:
+  workflow_dispatch:
+  schedule:
+    - cron: "17 2 * * *"
+  push:
+    branches:
+      - gaudi-ci
+
+jobs:
+  model-ci:
+    name: Model CI
+    uses: ./.github/workflows/self-scheduled-intel-gaudi.yml
+    with:
+      job: run_models_gpu
+      ci_event: Scheduled CI (Intel) - Gaudi3
+      runner_scale_set: itac-bm-emr-gaudi3-dell
+      slack_report_channel: "#transformers-ci-daily-intel-gaudi3"
+      report_repo_id: optimum-intel/transformers_daily_ci_intel_gaudi3
```

```suggestion
      report_repo_id: optimum-habana/transformers_daily_ci_intel_gaudi3
```
Maybe better in optimum-habana?

---

I'm not sure but should we use this then ? https://huggingface.co/Habana

---

Hmm after thinking about it more, let's keep it in optimum-intel. Optimum-habana will get merged into optimum-intel at some point anyway.